modules/javafx.controls/src/main/java/javafx/scene/control/skin/SliderSkin.java

```diff
@@ -389,6 +390,19 @@ public Object queryAccessibleAttribute(AccessibleAttribute attribute, Object...
                     cur.getX() - dragStart.getX() : -(cur.getY() - dragStart.getY());
             behavior.thumbDragged(me, preDragThumbPos + dragPos / trackLength);
         });
+
+        thumb.setOnMouseEntered(me -> {
+            if (getSkinnable().getTooltip() != null && getSkinnable().getTooltip().isAutoHide()) {
```

Comment:
Also, since the reason for this change is not immediately obvious, I'd add a comment explaining why (perhaps above tooltipConsumeAutoHidingEvents declaration, line 64?)

Reply:
Added above suggested edit and added comments